tests/utils/verification/volumeproperties.go

```diff
@@ -97,6 +100,75 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// GetContainerName returns full container name based on given short name
+func GetContainerName(hostName, shortName string) (string, error) {
+	cmd := dockercli.ListContainers + "--filter name='" + shortName + "' --format '{{.Names}}'"
+	return ssh.InvokeCommand(hostName, cmd)
+}
+
+// GetAllContainers returns all running containers on the give host based on the filtering criteria
+func GetAllContainers(hostName, filter string) (string, error) {
```

Comment:
I think `/volumeproperties.go` is not the better home to hold such utils. I would suggest to create another file or place into existing suitable place.